spec/requests/providers/online_bankings_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'does client use online banking requests', type: :request do
+  let(:applicant) { create :applicant }
+  let(:application) { create :legal_aid_application, applicant: applicant }
+  let(:application_id) { application.id }
+
+  describe 'GET /providers/applications/:legal_aid_application_id/applicant' do
+    let(:get_request) { get "/providers/applications/#{application_id}/does-client-use-online-banking" }
```

Comment:
I think `subject` makes more sense here. I know `<HTTP VERB>_request` is the pattern used across a lot of this code base, but it has problems - for example the headache of having to change all the instances of `post_request` to `patch_request` when the HTTP method changed. The fact that the name of the `describe` block is exactly the request specified in `get_request` implies that the request is the subject of the spec.